libraries/src/Table/Table.php

```diff
@@ -182,20 +287,39 @@ public function __construct($table, $key, DatabaseDriver $db, DispatcherInterfac
 		// Set the singular table key for backwards compatibility.
 		$this->_tbl_key = $this->getKeyName();
 
-		$this->_db = $db;
+		$this->_db = $db ?? Factory::getDbo();
```

Comment:
```suggestion
		$this->_db = $db ?? Factory::getContainer()->get(DatabaseDriver::class);
```

The `getDbo()` method will be deprecated in 5.0.